packages/strapi-plugin-content-type-builder/services/ContentTypeBuilder.js

```diff
@@ -84,7 +84,7 @@ module.exports = {
             newList = newList
               .delete(attrToRemoveInfos.index);
           } else {
-            const random = Math.floor(Math.random() * 1000); 
+            const random = Math.floor(Math.random() * 1000);
```

Comment:
Can you change this line with `const random = Math.random().toString(36).substring(7);`